### ISSUE TITLE
Pie Chart view as Int instead of double on segment?

### ISSUE THREAD

**Issue by Antuaaan** [danielgindi/Charts#2325]
Is it possible to toggle the label on the pie segments to be viewed as an integer(whole number) instead of with a decimal (double)? I am displaying an exact count of items so percentage won't work here but the .00 after each number doesn't look appealing so am trying to find  way to adjust it. Thank-you in advance to someone must have encountered this and will be kind enough to share with me the solution. 
<img width="258" alt="piechart" src="https://cloud.githubusercontent.com/assets/14073238/24657487/b29ca4fa-1946-11e7-8e77-0f94198ec7c7.png">

**TLDR** : Would rather have 5 and 6 in the view instead of 5.00 and 6.00

**Comment by liuxuan30** [comment]
use `valueFormatter`

**Comment by Antuaaan** [comment]
https://github.com/danielgindi/Charts/issues/1551 helped but thank you for the push in the right direction. 

`let format = NumberFormatter()`
`format.numberStyle = .none`
`let formatter = DefaultValueFormatter(formatter: format)`
`pieChartData.setValueFormatter(formatter)`

**Comment by liuxuan30** [comment]
more info: it's protocol, so it's very flexible, and the default ones might have bugs.
